osquery/tables/system/windows/processes.cpp

```diff
@@ -235,12 +235,28 @@ void genProcess(const WmiResultItem& result, QueryData& results_data) {
   auto ret = OpenProcessToken(hProcess, TOKEN_READ, &tok);
   if (ret != 0 && tok != nullptr) {
     unsigned long tokOwnerBuffLen;
-    ret = GetTokenInformation(tok, TokenOwner, nullptr, 0, &tokOwnerBuffLen);
+    ret = GetTokenInformation(tok, TokenUser, nullptr, 0, &tokOwnerBuffLen);
     if (ret == 0 && GetLastError() == ERROR_INSUFFICIENT_BUFFER) {
       tokOwner.resize(tokOwnerBuffLen);
       ret = GetTokenInformation(
-          tok, TokenOwner, tokOwner.data(), tokOwnerBuffLen, &tokOwnerBuffLen);
+          tok, TokenUser, tokOwner.data(), tokOwnerBuffLen, &tokOwnerBuffLen);
     }
+
+	// Check if the process is using an elevated token
+	BOOL elevated = FALSE;
+	TOKEN_ELEVATION Elevation;
+	DWORD cbSize = sizeof(TOKEN_ELEVATION);
```

Comment:
I think this and the `elevated` variable above could be an `auto` type, would you mind changing them?

Reply:
Compiler didn't like cbSize declared as auto.